peripherals/rcc/rcc_l4.yaml

```diff
@@ -13,3 +45,44 @@ RCC:
       Range24M: [0b1001, "range 9 around 24 MHz"]
       Range32M: [0b1010, "range 10 around 32 MHz"]
       Range48M: [0b1011, "range 11 around 48 MHz"]
+
+  CCIPR:
+    ADCSEL:
+      NoClock: [0, "No clock selected"]
+      PLLSAI1: [1, "PLLSAI1 clock selected"]
+      PLLSAI2: [2, "PLLSAI2 clock selected"]
```

Comment:
PLLSAI2 option is not valid for RM0394 parts (STM32L41x/L42x/L43x/L44x/L45x/L46x)

An "only for" comment in the description is OK to explain this